storage/innobase/os/os0file.cc

```diff
@@ -4771,31 +4790,100 @@ readahead requests. */
 
 #if defined(LINUX_NATIVE_AIO)
 /*******************************************************************//**
+Submit buffered AIO requests on the given segment to the kernel. */
+UNIV_INTERN
+void
+os_aio_linux_dispatch_read_array_submit()
```

Comment:
The submission API (should_buffer arg everywhere) is platform-independent, so the "flush" API should not be either, please remove "linux" from name, push #ifdef inside, remove #ifdefs from callers

Reply:
done